lib/web/join_tokens.go

```diff
@@ -630,13 +630,20 @@ func getJoinScript(ctx context.Context, settings scriptSettings, m nodeAPIGetter
 	var buf bytes.Buffer
 	// If app install mode is requested but parameters are blank for some reason,
 	// we need to return an error.
+	var appServerResourceLabels []string
```

Comment:
Suggestion: move this declaration up so that the comment about app install mode stays next to the code that checks `settings.appInstallMode`.

```suggestion
	var buf bytes.Buffer, appServerResourceLabels []string
	// If app install mode is requested but parameters are blank for some reason,
	// we need to return an error.
```